### ISSUE TITLE
VSCodium says update is available after being updated

### ISSUE THREAD

**Issue by mdavis199** [VSCodium/vscodium#222]
**Describe the bug**
Just updated to release 1.36.0 on Windows 10.  Codium says there is an update available, and downloads v 1.36.0 again.

**Please confirm that this problem is VSCodium-specific**
- [x] This bug doesn't happen if I use Microsoft's Visual Studio Code. It only happens in VSCodium.

**Please confirm that the issue/resolution isn't already documented**
- [x ] I checked [the Docs page](https://github.com/VSCodium/vscodium/blob/master/DOCS.md) and my issue is not mentioned there.

Related to #185 (which was for version 1.35) and maybe #192 

**To Reproduce**
- Download the 1.36 release. 
- Extract over existing vscodium folder.  
- Launch vscodium
- VScodium says a download is available and offers up another copy of the same update.

**Expected behavior**
VSCodium should recognize that it is the latest release

**Desktop (please complete the following information):**
 - OS: Windows 10 Enterprise, 1809
 - Architecture: x64
 - Version 1.36.0

**Comment by stripedpajamas** [comment]
Ok, something awry with the version JSON file again. Ran versions/integrity.js and got this result:

```
{
    "win32Archive": "Hashes match",
    "win64Archive": "Hashes match",
    "win32User": "Invalid hashes",
    "win64User": "Invalid hashes",
    "darwin": "Hashes match",
    "linux32": "Hashes match",
    "linux64": "Hashes match"
}
```

Will investigate why the two Windows user install hashes don't match what's in the version JSON file.

**Comment by stripedpajamas** [comment]
For now I've manually updated the user install hashes in the versions repo. Can you see if it's still stuck in the infinite update loop?

**Comment by linsui** [comment]
Yeah, still stuck.

**Comment by stripedpajamas** [comment]
I have spun up a Windows machine with VSCodium 1.35.x installed and it autoupdated to 1.36.1 (released today) and I am not stuck in any kind of update loop.

@mdavis199 @linsui can you confirm this is still an issue?

**Comment by mdavis199** [comment]
- v 1.36.0 says one update available
- I downloaded 1.36.1 and extracted files into the vscodium folder to overwrite existing files
- launched codium
- and, it reports "There are no updates available"
Success.

**Comment by linsui** [comment]
It seems that this problem happens on 1.37.1 again 😄

**Comment by stripedpajamas** [comment]
Am seeing proper hashes in the versions repo:

```
Summary:
{
    "win32Archive": "Hashes match (1.37.1)",
    "win64Archive": "Hashes match (1.37.1)",
    "win32User": "Hashes match (1.37.1)",
    "win64User": "Hashes match (1.37.1)",
    "darwin": "Hashes match (1.37.1)",
    "linux32": "Hashes match (1.35.1)",
    "linux64": "Hashes match (1.37.1)"
}
```

**Comment by stripedpajamas** [comment]
@linsui what is the experience ? continual updating? specifically what variant are you using (win32 / win64 user/system)?

**Comment by linsui** [comment]
VSCodium-win32-x64-1.37.1.zip user, continual updating.

**Comment by TroySchmidt** [comment]
Same thing here.  v1.37.1 (user setup) Windows_NT x64 10.0.18362

**Comment by stripedpajamas** [comment]
The issue is that the commit hash baked into the product.json file for the Windows assets is `9afbbd903b7b481aa127d4d8d68626397e6a5135`, which is a VSCodium commit hash, not a vscode commit hash. We have seen this before but maybe the commit logic on the MS side has changed. I will investigate.

Regardless, we will not want to republish Windows assets as that causes downstream issues (#192). I will update the version JSON file to use the VSCodium hash for Windows assets and that should fix the update loop.

**Comment by stripedpajamas** [comment]
Have merged in https://github.com/VSCodium/versions/commit/cf65fa23658c87badac7eefa3fe60cc03b91ae75

Can someone try updating and then confirming that it doesn't prompt afterwards? @linsui @TroySchmidt

**Comment by TroySchmidt** [comment]
@stripedpajamas that fixed it!

**Comment by stripedpajamas** [comment]
Awesome, thanks for checking @TroySchmidt 

I will merge in the fix #244 so hopefully this doesn't happen with 1.38.0 😃
